_posts/newsletters/2022-11-04-newsletter.md

```diff
@@ -0,0 +1,261 @@
+    
+    
+<a name ="jobarchive"></a>
+# **Help Create an RSE Job Description Archive**
+    
+Following the recent [ADSA/US-RSE Academic Data Scientist/RSE Career Support Workshop](https://academicdatascience.org/adsa-meetings/2022-fall-workshop) that is writing a handbook on hiring and supporting DSs/RSEs at academic institutions, we would like to archive the full postings for jobs listed on the [US-RSE Job Board](https://us-rse.org/jobs/) on a regular basis.  This archive would help people writing job descriptions for their own institutions, and it could also help applicants to see the range of RSE positions available.  If you think you could set up an automated system to periodically collect the original full postings from company/university sites and archive them, please get in touch with Christina Maimone via [email](mailto:christina.maimone@northwestern.edu) or on Slack.
+
+<a name="sc-update"></a>
+# **Steering Committee Recap**
+
+Since August, the Steering Committee has been discussing and working on the following topics:
+
+ * The first US-RSE Conference in October 2023: more details in this newsletter
+ * What funding US-RSE needs to grow as a community and what paid positions US-RSE needs to be successful, such as a Community Manager
+ * Improving the Code of Conduct reporting and response processes (there's a working group for anyone interested)
+ * Distribution of US-RSE swag and options for opening a US-RSE store again this year for fundraising
+ * Community events and virtual talk series
+ * Governance of US-RSE, including issues such as: the structure of, and communication between, working groups and whether an advisory board separate from the Steering Committee would be useful
+    
+
+
+
+<a name="news"></a>
+# **Community News**
+
+* A new RSE association has been formed in Asia! We in US-RSE are looking forward to international collaboration with the [RSE Asia Association](https://rse-asia.github.io/RSE_Asia/)!
+* [ACCESS](https://www.nsf.gov/news/special_reports/announcements/042222-access.jsp) - the successor of XSEDE - started, and you can apply with research projects for large HPC resources. [https://support.access-ci.org/matchplus](https://support.access-ci.org/matchplus)
+* [CI CoE: SGX3 - A Center of Excellence to Extend Access, Expand the Community, and Exemplify Good Practices for CI Through Science Gateways](https://www.nsf.gov/awardsearch/showAward?AWD_ID=2231406) was just funded for five years and is extending the work that started with the Science Gateways Community Institute. [SGX3](https://sciencegateways.org/) guides the annual Gateways conference series, offers Focus Weeks that provide training on sustainability of research projects, provides Gateways Central that will be a focal point of exchanging knowledge, experience and technology of science gateway solutions and frameworks and Blueprint Factories target different topics and technologies.
```

Comment:
Not sure if a late comma or more is needed here, by the end of the last sentence I am losing track of the grammar and significance. Does SGX3 provide Blueprint Factories or are we just saying something about what they do? Though presumably the solutions and frameworks also target different topics and technologies? Not a hard refusal, just a "you're losing me here".

Reply:
Yes - It got a bit much. How would be "[SGX3](https://sciencegateways.org/) guides the annual Gateways conference series, offers Focus Weeks that provide training on sustainability of research projects, provides Gateways Central that will be a focal point of exchanging knowledge and technology of science gateway solutions  as well as creates Blueprint Factories that target different topics and technologies."

Reply:
I think that's good, but I'd add a comma and use "and" instead of "as well as." I'll make the suggested change